exercises/concept/lasagna/.docs/introduction.md

```diff
@@ -3,122 +3,120 @@
 The entire Julia track will require you to treat your solution like small libraries, i.e. you need to define functions, types etc. which will then be run against a test suite.
 For that reason, we will introduce named functions as the very first concept.
 
-## Functions
+Julia is a dynamic, strongly-typed programming langauge.
+The programming style is mainly functional, though with more flexibility than in languages such as Haskell.
 
-### Defining functions
+## Comments
```

Comment:
This section describes how to make comments, but not what they are. The previous section mentions functions being the next concept so going immediately into comments is a bit confusing. This exercise also doesn't involve working with comments beyond noting what needs to be implemented so I'm not convinced it's necessary. Named functions and constants seem to be the focus.

Reply:
Agree

Reply:
Removed the entire subsection.